outlines/text/json_schema.py

```diff
@@ -203,7 +203,7 @@ def match_step_to_regex(step):
     """
     match step:
         case str() as step:
-            return step
+            return step.replace("{", "\\{").replace("}", "\\}")
```

Comment:
Will [`re.escape`](https://docs.python.org/3/library/re.html#re.escape) work for this?

Reply:
It's actually a much better solution.